misc/logging/src/lib.rs

```diff
@@ -0,0 +1,12 @@
+use tracing_subscriber::EnvFilter;
```

Comment:
```suggestion
pub use tracing_subscriber::EnvFilter;
```

Re-export `EnvFilter` so that no additional dependency to `tracing_subscriber` is needed when using `init_tracing_subscriber_with_env_filter`?

Reply:
done